ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.8__okd.yaml

```diff
@@ -0,0 +1,99 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: origin
+    tag: base
+  ocp_builder_rhel-7-golang-1.13:
+    name: builder
+    namespace: ocp
+    tag: rhel-7-golang-1.13
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.7
+  ocp_builder_rhel-8-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```

Comment:
shouldn't it be 1.15?

Reply:
I'm using the same tags as in openshift-cluster-samples-operator-release-4.8.yaml here